config/lint.xml

```diff
@@ -62,4 +62,5 @@
 
     <!-- This flags IDs used via View Binding: https://issuetracker.google.com/issues/170658134 -->
     <issue id="UnusedIds" severity="ignore" />
+    <issue id="Range" severity="ignore" />
```

Comment:
Could you add a comment explaining this? I'm guessing it's to do with `Cursor` indices but good to make sure it's clear here for the future.

Reply:
> I'm guessing it's to do with Cursor indices. 

Yes. I added a link to a discussion on stackoverflow.